sdk/eventhub/event-hubs/src/eventDataBatch.ts

```diff
@@ -83,6 +97,7 @@ export class EventDataBatch {
    * @returns A boolean value indicating if the event data has been added to the batch or not.
    */
   public tryAdd(eventData: EventData): boolean {
+    throwTypeErrorIfParameterMissing(this._context.connectionId, "eventData", eventData);
```

Comment:
If user don't pass the eventData in tryAdd(), we should throw parameter missing error or silently return false?
cc:  @ramya-rao-a @chradek

Reply:
we should throw parameter missing.